openpype/pipeline/colorspace.py

```diff
@@ -589,3 +589,61 @@ def _get_imageio_settings(project_settings, host_name):
     imageio_host = project_settings.get(host_name, {}).get("imageio", {})
 
     return imageio_global, imageio_host
+
+def get_display_view_colorspace_name(config_path, display, view):
+    """get view colorspace name for the given display and view.
+
+    Args:
+        config_path (str): path string leading to config.ocio
+        display (str): display name e.g. "ACES"
+        view (str): view name e.g. "sRGB"
+
+    Returns:
+        view color space name (str) e.g. "Output - sRGB"
+    """
+
+    if not compatibility_check():
+        # python environment is not compatible with PyOpenColorIO
+        # needs to be run in subprocess
+        return get_display_view_colorspace_subprocess(config_path,
+                                                      display, view)
+
+    from openpype.scripts.ocio_wrapper import _get_display_view_colorspace_name #noqa
```

Comment:
Shouldn't the implementation of `_get_display_view_colorspace_name` be here and import it inside the scripts instead? Like the scripts are not meant to be source of functions.

Reply:
I thought this was the convention because it's done the same way in another two functions.

https://github.com/ynput/OpenPype/blob/0c3fa2b61079e0497bccc9ae88e381f681afe39e/openpype/pipeline/colorspace.py#L272

https://github.com/ynput/OpenPype/blob/0c3fa2b61079e0497bccc9ae88e381f681afe39e/openpype/pipeline/colorspace.py#L308

if the implementation is moved out to `colospace.py` 
then should I do this in `ocio_wrapper`

```python
def  get_display_view_colorspace_name (in_path, out_path, display, view):

    json_path = Path(out_path)
    from openpype.pipeline.colorspace import _get_display_view_colorspace_name

    out_data = _get_display_view_colorspace_name(in_path, display, view)

    with open(json_path, "w") as f:
        json.dump(out_data, f)

    print(f"Viewer data are saved to '{json_path}'")
```